### PR TITLE
cli that works with published packages

### PR DESCRIPTION
**Pull request by in4margaret**
to run: 
same steps as before, but project creation changed to the line below, where --windowsVersion is teh version of react-native-windows package

react-native windows --template vnext --windowsVersion 0.58.0-vnext.31 --verbose

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/Microsoft/react-native-windows/pull/2302)

### CODE REVIEW

**Comment by acoates-ms** [comment]
For a brand new app, can they leave of the windowsVersion and get the latest version published as vnext?
I think by default we want to make sure people get the latest.

**Comment by harinikmsft** [comment]
To confirm the steps for installing - (after installing dependencies like node.js, yarn, react-native)
```
react-native init <projname>
cd <projname>
yarn add rnpm-plugin-windows
yarn link rnpm-plugin-windows
react-native windows --template vnext
```
And then to run the solution:
```
react-native run-windows
npm run start
Open project in VS, F5
```

Sound correct?

**Comment by in4margaret** [comment]
@acoates-ms  Added retrieval of the latest vnext version. @harinikmsft  '
* react-native run-windows 
OR
* npm run start  and Open project in VS, F5'

File: RNWCS/local-cli/rnpm/windows/src/windows.js [hunk 1]
```diff
@@ -21,98 +20,75 @@ const REACT_NATIVE_WINDOWS_GENERATE_PATH = function() {
...
-    'react-native-windows',
-    'RNWCPP',
-    'local-cli',
-    'generate-windows.js'
+/**
+ * Returns latest version of react-native-windows where version contains 'vnext'
+ * @param {'yarn'|'npm'} pkgmgr
+ * @param {boolean} isYarn
+ */
+const getLatestVnextVersion = function (pkgmgr, isYarn) {
```
Comment (acoates-ms) [inline RNWCS/local-cli/rnpm/windows/src/windows.js]: For npm you can call:
`npm info react-native-windows@vnext version` -- And that should give you the version you want.

For yarn, I think you want:
`JSON.parse(exec('yarn info react-native-windows --silent'))['dist-tags'].vnext;`

**Comment by acoates-ms** [comment]
It also looks like you need to run yarn to update the lock file for the new dependencies.

**Pull request merged**